src/store/actions/index.js

```diff
@@ -0,0 +1 @@
+import foodActions from './food';
```

Comment:
You already have file with named exports, this file is unused. Please remove it

Reply:
Remove file food.js